test/unit/server/UnixSocketHttpServerFactory.test.ts

```diff
@@ -0,0 +1,102 @@
+import type { Server } from 'http';
```

Comment:
This should be part of the `BaseHttpServerFactory.test.ts` test suite. Unit tests should have a 1-to-1 mapping to existing files.